modules/nf-core/mafft/tests/main.nf.test

```diff
@@ -0,0 +1,214 @@
+nextflow_process {
+
+    name "Test Process MAFFT"
+    script "../main.nf"
+    process "MAFFT"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "mafft"
+
+    // test("SARS-CoV-2 scaffolds fasta") {
+    //     when {
+    //         process {
+    //             """
```

Comment:
why are these tests commented out ? @Joon-Klaps

Reply:
Was for a faster update snapshot and forgot to uncomment as I had been trying multiple ways to do it